torchci/lib/GeneralUtils.ts

```diff
@@ -10,8 +10,6 @@ export function includesCaseInsensitive(
 }
 
 export const fetcher = (url: string) => fetch(url).then((res) => res.json());
-export const fetcherWithToken = (url: string, token: string) =>
-  fetch(url, { headers: { Authorization: token } }).then((res) => res.json());
```

Comment:
Only used in WorkflowDispatcher, and we don't actually return any json from the route there so it was erroring

Also not sure how it was working before since the input should be an array